front/src/api/Genres.api.ts

```diff
@@ -0,0 +1,14 @@
+import GigzFetcher from '../services/GigzFetcher';
+import GigzResponse from '../types/GigzResponse';
+
+export interface Genre {
+  name: string;
+  color: string;
+  id: string;
+}
```

Comment:
Maybe we can add this to the type's folder, as it will be used in different components

Reply:
Good point, I think I did the same thing in `types` folder